guides/micronaut-mongodb-asynchronous/micronaut-mongodb-asynchronous.adoc

```diff
@@ -131,19 +131,25 @@ include::{commondir}/common-runapp.adoc[]
 
 [source, bash]
 ----
-% curl -d '{"name":"Pear"}' 
-    -H "Content-Type: application/json" 
-    -X POST http://localhost:8080/fruits
+curl -d '{"name":"Pear"}'
+     -H "Content-Type: application/json"
+     -X POST http://localhost:8080/fruits
+----
+
+[source, bash]
+----
+curl -i localhost:8080/fruits
+----
 
-% curl -i localhost:8080/fruits                                                                     
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks